tests/autodiff/material2/MxWeights.slang

```diff
@@ -3,6 +3,11 @@
 public struct MxWeights<let TBsdfCount : int>
 {
     public float3 weights[TBsdfCount];
+
+    public __init()
+    {
+        weights[0] = float3(0.0f);
```

Comment:
Shouldn't you do for-loop?
```
for (int i = 0; i < TBsdCount; i++)
weights[i] = float3(0);
```

Reply:
Yes, it should be. I had it like this because it was enough to silence the warnings (partial uninitialization is not checked for yet).